storage/migrations/02_runtimes.up.sql

```diff
@@ -129,10 +129,10 @@ CREATE TABLE chain.runtime_events
   evm_log_signature BYTEA CHECK (octet_length(evm_log_signature) = 32)
 );
 CREATE INDEX ix_runtime_events_round ON chain.runtime_events(runtime, round);  -- for sorting by round, when there are no filters applied
-CREATE INDEX ix_runtime_events_tx_hash ON chain.runtime_events(tx_hash);
-CREATE INDEX ix_runtime_events_tx_eth_hash ON chain.runtime_events(tx_eth_hash);
+CREATE INDEX ix_runtime_events_tx_hash ON chain.runtime_events/*USING hash */(tx_hash); -- updated in 13_runtime_indexes.up.sql
```

Comment:
is this the first we're using /* ... */ comments? does is this postgresql specific syntax?

Reply:
tests don't complain about it :tipping_hand_person: